File: C#/Day8DictionariesAndMaps.cs
using System.Collections.Generic;
using System;

class Solution
{
    static void Main(String[] args)
    {
        var n = Convert.ToInt32(Console.ReadLine());
        string name;

        Dictionary<string, int> phoneBook = new Dictionary<string, int>();

        for (int i = 0; i < n; i++)
        {
            var pair = Console.ReadLine().Split(' ');
            phoneBook[pair[0]] = Convert.ToInt32(pair[1]); // or phoneBook.Add(pair[0], Convert.ToInt32(pair[1]));
        }

        while ((name = Console.ReadLine()) != null)
        {
            if (phoneBook.ContainsKey(name))
                Console.WriteLine($"{name}={phoneBook[name]}");
            else
                Console.WriteLine("Not found");
        }
    }
}

File: Java/Day8DictionariesAndMaps.java
import java.util.*;

class Solution{
    public static void main(String []argh){
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        
        HashMap<String, Integer> phoneBook = new HashMap<String, Integer>();
        
        for(int i = 0; i < n; i++){
            String name = in.next();
            int phone = in.nextInt();
            phoneBook.put(name, phone);
        }
        
        while(in.hasNext()){
            String name = in.next();
            
        	if (phoneBook.containsKey(name)) {
        		System.out.println(name + "=" + phoneBook.get(name));
        	} else {
        		System.out.println("Not found");
            }
        }
        
        in.close();
    }
}